### PR TITLE
Fix Word converter warnings

### PR DESCRIPTION
**Pull request by jskeet**
Please verify these changes carefully.

With these in place, the only warnings are:

```text
classes.md(2030): WARNING MD16: Term 'block body' defined a second time
classes.md(2024): WARNING MD16b: Here was the previous definition of term 'block body'
classes.md(2030): WARNING MD16: Term 'expression body' defined a second time
classes.md(2024): WARNING MD16b: Here was the previous definition of term 'expression body'
classes.md(3171): WARNING MD16: Term 'expression body' defined a second time
classes.md(2024): WARNING MD16b: Here was the previous definition of term 'expression body'
```

Those can be addressed later, as noted in #948.

After these changes are applied, this is the result: [standard.docx](https://github.com/dotnet/csharpstandard/files/12681507/standard.docx)

### CODE REVIEW

File: standard/grammar.md [hunk 1]
```diff
@@ -10,15 +10,15 @@ This annex contains the grammar productions found in the specification, includin
 
 ```ANTLR
 
-// Source: §6.3.1 General
+// Source: �6.3.1 General
```
Comment (KalleOlaviNiemitalo) [inline standard/grammar.md]: Why did this change?

Comment (jskeet) [reply standard/grammar.md]: Ooh, that's weird. grammar.md was updated by the tooling rather than by hand. Will check it out...

Comment (jskeet) [reply standard/grammar.md]: Got it - it was an encoding issue when running bash on Windows. Fixed in the GetGrammar tool.

**Comment by jskeet** [comment]
New version of the Word doc: 
[standard.docx](https://github.com/dotnet/csharpstandard/files/12683632/standard.docx)

**Comment by KalleOlaviNiemitalo** [comment]
Out of curiosity, how does the C# 7 standardization proceed from here?  I see there is a TC49-TG2 meeting scheduled for 4 October 2023; is that when you'll formally decide whether to send the Word document to ECMA?

**Comment by jskeet** [comment]
> Out of curiosity, how does the C# 7 standardization proceed from here? I see there is a TC49-TG2 meeting scheduled for 4 October 2023; is that when you'll formally decide whether to send the Word document to ECMA?

Nope, that was basically decided yesterday. The October meeting is to make a start on the pre-C#-8 work, and to discuss how to divide the C# 8 features.

Rex is going to perform tweaks on the Word doc (there are bits that the converter can't do) and send it on to ECMA in about a week, to start the ratification process.

**Review by BillWagner — APPROVED** [review]
I checked this carefully, and I say let's :shipit:

I don't think the changes in grammar.md are strictly necessary, as they'll be rewritten by an automated PR, but they are consistent with the changes in other files.

**Comment by jskeet** [comment]
> I don't think the changes in grammar.md are strictly necessary, as they'll be rewritten by an automated PR, but they are consistent with the changes in other files.

They were all automated already, so should be okay :)

**Comment by jskeet** [comment]
Ah - it looks like we need 3 approving reviews for standard- branches. Let's wait until @RexJaeschke reviews as well, but at that point we can override and merge :)

**Review by RexJaeschke — APPROVED** [review]

**Pull request merged**